html5/_main_toc.html.slim

```diff
@@ -189,6 +189,9 @@ nav#toc.collapse.bd-links.toc
           a href="https://docs.killbill.io/latest/Kill-Bill-Glossary.html"
             | Glossary
   ul.nav.navbar-nav
+        li
+          a href="https://docs.killbill.io/latest/Kill-Bill-Glossary.html"
+            | Glossary
     li
```

Comment:
There is an extra Glossary link. This messes up the build, I had deleted it earlier but looks like you have an older version of this file. Can you please delete it?

Reply:
@reshmabidikar - Fixed...please check